src/mlpro/bf/examples/howto_bf_023_normalizers.py

```diff
@@ -0,0 +1,126 @@
+ ## -------------------------------------------------------------------------------------------------
+## -- Project : MLPro - A Synoptic Framework for Standardized Machine Learning Tasks
+## -- Package : mlpro.bf.examples
+## -- Module  : howto_bf_023_normalizers.py
+## -------------------------------------------------------------------------------------------------
+## -- History :
+## -- yyyy-mm-dd  Ver.      Auth.    Description
+## -- 2022-09-16  0.0.0     LSB      Creation
+## -- 2022-09-25  1.0.0     LSB      Release of first version
+## -------------------------------------------------------------------------------------------------
+
+"""
+Ver. 1.0.0 (2022-09-25)
+Example file for demonstrating the use of MLPro's normalizer for normalizing and de-normalizing data.
+"""
```

Comment:
Could you please add a section "You will learn:" like in howto_bf_011_event_handling.py? But please beware: the blank lines are necessary there because of RTD formatting.